src/legacy-styles/components/_editor-override.scss

```diff
@@ -64,6 +63,9 @@
 		p + p {
 			text-indent: 2rem;
 		}
+		p + p > * {
+			text-indent: 0;
+		}
```

Comment:
I'm considering removing all text indents tbh. There have been several people suggesting removing them, and they only exist to imitate the source books.